connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -35,7 +35,7 @@ message Relation {
     Project project = 3;
     Filter filter = 4;
     Join join = 5;
-    Union union = 6;
```

Comment:
Why is Set better than Union? Just from a SQL naming persepective, Union might be what users are used to?

Reply:
I am pretty sure`Set operation` is the SQL word to describe `union` `intersect` `except`. But Wenchen suggests to have individual plans for each, in that it won't matter.